test/CMakeLists.txt

```diff
@@ -5,6 +5,12 @@
 set(CMAKE_CXX_STANDARD 17)
 set(CMAKE_CXX_STANDARD_REQUIRED YES)
 
```

Comment:
it's this time of the year - pls updates copyright dates

Reply:
Updated, thanks